### ISSUE TITLE
[BUG] or [Change] ? - Trap-door has creating collision after a few updates.

### ISSUE THREAD

**Issue by nokturnihs** [tsunamayo/Starship-EVO#2417]
### Short Description of the issue:

So the ship I submitted - over on the backer github - uses a trapdoor that an elevator passes through (when open) to serve as the only means of access onto the ship. Worked fine when submitted, a few updates after that period of time, but somewhere along the lines, the trapdoor must have received some changes, because now it causes the elevator to stick as though there's a block preventing it's movement (collision) - it's not the slide-rail mechanism - if you remove the trapdoor the lift functions without issue. I don't know if this is a bug or an intentional change - if the change was to reduce performance issues, cool - i can change the doors. if it's a bug, just wanted to let you know.  re-added the blueprint for convenience.
[Nox Imperium Enterprises Cerberus Transport.zip](https://github.com/tsunamayo/Starship-EVO/files/4768995/Nox.Imperium.Enterprises.Cerberus.Transport.zip)

### Steps to reproduce the issue:

...

### Environment
- Starship EVO version: 
- Graphics Card: 

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

**Comment by Sullos** [comment]
I reported this bug myself, as trapdoors aren't meant to be able to passed through if closed. But sounds like by adding the collision to the closed door, its also there even when its open.

**Comment by nokturnihs** [comment]
Oh - looked for a trapdoor report but I missed it. Sorry @tsunamayo   !
Yeah @Sullos it could be related to small brick collision too I suppose? Around the same time I noticed occasionally getting "bumped" by bricks on child entities and/or clipping through small brick floors for brief periods of time (I'm assuming during a collision/rendering update). Either way...

**Comment by Sullos** [comment]
So the original bug report I made was https://github.com/tsunamayo/Starship-EVO/issues/2324 . Whilst the majority of that report is focussed on the broken textures and such, before we got the broken textures, I'd been walking around in jetpack mode on my ship and fell into one of my external storage compartments. I'm guessing when the collision mesh for it was updated so you couldn't fall through it, it wasn't applied to the lid of the trapdoor specifically? Just an idea.
